package.json

```diff
@@ -31,47 +31,46 @@
   "homepage": "https://github.com/esbb48/Kizuna#readme",
   "dependencies": {
     "babel-polyfill": "^6.9.1",
-    "grommet": "^0.6.10",
+    "extract-text-webpack-plugin": "^v2.0.0-beta.4",
+    "grommet": "^1.0.1",
     "react": "^15.3.0",
     "react-dom": "^15.3.0",
-    "react-hot-loader": "^3.0.0-beta.1",
+    "react-hot-loader": "^3.0.0-beta.6",
```

Comment:
okay, 

for last pr, it will caused error.
but for now. Change to pu on devDependencies. It works.